style.css

```diff
@@ -0,0 +1,61 @@
+body {
+    background-color: #fff;
+    font-family: 'Roboto', sans-serif;
+    letter-spacing: 0.5px;
+    height: 100%;
+  }
+  
+  header {
+    display: flex;
+    flex-direction: row;
+  }
+  
+  #brand {
+    margin-left: 20px;
+  }
+  .youtube {
+    height: auto;
+    width: 30px;
+  }
+  
+  .form-inline {
+    display: flex;
+    align-items: center;
+    margin-left: 470px;
+    /* margin-right: -200px; */
+  }
+  
+  .input-group {
+    width: 100%;
+    height: 70px;
+  }
+  
+  .youtube {
+    text-decoration: none;
+  }
+  .search-form {
+    border-radius: 30px 0 0 30px;
+  }
+  
+  .input-group-btn {
+    max-width: 90px;
+  }
+  
+  .search-btn {
+    color: rgb(26, 26, 36);
```

Comment:
Why don't you try the following:

- [ ] To increase the width of the search bar so that it is long enough and it looks nice to emulate the original youtube page.
- [ ] To add the icons to the right of the search bar so that the 'bell, video, and user' icons show, this way the youtube website will be emulated. 
- [ ]  to also maintain the shades of white in the search bar and search icon, also the entire body and the description section. This will result in a more convenient user experience since sections(including buttons) of text or video can be separated visually using the different shades of white.
@ABDULALI3468